.github/workflows/check-commit-message.yml

```diff
@@ -0,0 +1,38 @@
+name: check-commit-message
+
+run-name: Check commit messages by ${{ github.actor }} on ${{ github.ref }}
+on:
+  push:
+    branches:
+      - "**"
+  pull_request:
+    branches:
+      - master
+
+jobs: 
+  # push events have commits as a list of actions
+  check_commit_on_push:
+    if: github.event_name == 'push'
+    name: check commit message
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Is this necessary?

Reply:
I don't really see how to access the python file otherwise..